vignettes/vignette.Rmd

```diff
@@ -1,64 +1,72 @@
 
 ```{r load_packages, message = FALSE}
-#install.packages(c("dplyr", "palaeoverse", "readr", "ggplot2", "deeptime", "rnaturalearth", "rnaturalearthdata", "rgplates", "fossilbrush", "CoordinateCleaner"))
+# install.packages(c("CoordinateCleaner", "deeptime", "dplyr", "fossilbrush", "ggplot2", "palaeoverse", "readr", "rgplates", "rnaturalearth", "rnaturalearthdata"))
+library(CoordinateCleaner)
+library(deeptime)
 library(dplyr)
+library(fossilbrush)
+library(ggplot2)
 library(palaeoverse)
 library(readr)
-library(ggplot2)
-library(deeptime)
+library(rgplates)
 library(rnaturalearth)
 library(rnaturalearthdata)
-library(rgplates)
-library(fossilbrush)
-library(CoordinateCleaner)
 ```
 
 ## Rule 1: Choose the right data for your question
 
-In this example, we will explore, clean and interrogate the fossil record of crocodiles. As ectotherms, crocodiles are highly reliant on the environment in which they live in order to maintain a functional internal body temperature. Because of this, their spatial distribution is constrained to warm climates, so their fossil record is commonly used as an indicator of palaeoclimate. Here we will investigate the palaeodiversity of crocodiles, with the specific goal of reconstructing their biogeographic patterns throughout the Paleogene. The data will be sourced from the [Paleobiology Database](https://paleobiodb.org).
+In this example, we are interested in the fossil record of crocodiles. As ectotherms, crocodiles are highly reliant on the environment in which they live in order to maintain a functional internal body temperature. Because of this, their spatial distribution is constrained to warm climates, so their fossil record is commonly used as an indicator of palaeoclimate. Here we will investigate the palaeodiversity of crocodiles, with the specific goal of reconstructing their latitudinal range throughout the Paleogene.
 
-## Rule 2: Keep raw data raw
+To meet this goal, we need to acquire occurrence data for fossil crocodiles throughout the Paleogene. Initially we have decided not to place further taxonomic constraints on our search, so we will include all occurrences belonging to the order 'Crocodylia'. We are interested in the clade's biogeography, so we will need all occurrences globally, and we need to ensure that we have geographic coordinates associated with our occurrences.
 
-<!--# Would it be better to download directly from the API and show the first step as saving the file? -->
+We will turn to one of the largest sources of fossil occurrence data, the [Paleobiology Database](https://paleobiodb.org). We opted to download our data by completing the 'Download' form on the Paleobiology Database website. Practically, this uses the options entered into the form to create a call to the database's API (application programming interface), and then enacts this call. More information about the API service can be found [here](https://paleobiodb.org/data1.2/).
```

Comment:
In the html file, the links look perfect, but in the pdf version it is not always obvious when we have a link. I would suggest forcing links to be bold, underlined, or similar. If having the pdf is causing too many issues, I think it is also fine for us to just include the rmd file, and deploy the ten rules html file to our website (e.g. palaeoverse.org/ten-rules) or similar.

Reply:
FYI the links in the pdf are now blue